src/test/java/com/everis/control/CarServiceTest.java

```diff
@@ -0,0 +1,95 @@
+package com.everis.control;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import com.everis.entity.Car;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CarServiceTest {
+
+	@InjectMocks
+	CarService carService;
+
+	@Mock
+	private CarDao carDao;
+
+	@Test
+	public void testGetCars() {
+
+		try {
+			final List<Car> allcars = new ArrayList<>();
+			Mockito.when(carDao.findAll()).thenReturn(allcars);
+			final List<Car> cars = carService.getCars();
+			Assert.assertEquals(allcars, cars);
+		} catch (Exception e) {
+			Assert.fail();
+		}
+	}
+
+	@Test
+	public void testGetCar() {
```

Comment:
Is this test working ? Did you try to run it ?

Reply:
yes its works